### ISSUE TITLE
Store private key material in a more secure place within an OS user account

### ISSUE THREAD

**Issue by vitropy** [entropyxyz/cli#110]
Consider putting the private key material generated for an account into the OS-provided credential store, perhaps using a cross platform abstraction such as [keytar](https://www.npmjs.com/package/keytar).

Currently, private key data generated by the CLI is placed into a plain text, JSON formatted configuration file.

**Comment by mixmix** [comment]
YES! this has been discussed, and we're keen to move towards this.
Good to have an issue to track it

**Comment by mixmix** [comment]
I'm copying your message over into other thread and closing this one 🤦
